test/unit/metrics/test_plot_metricframe.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+import pytest
+from sklearn.metrics import recall_score, accuracy_score, confusion_matrix
+from fairlearn.experimental.enable_metric_frame_plotting import plot_metric_frame
+import matplotlib
+
+from fairlearn.metrics import MetricFrame
+from fairlearn.metrics._plotter import (
+    _GIVEN_BOTH_ERRORS_AND_CONF_INT_ERROR,
+    _METRIC_AND_ERRORS_NOT_SAME_LENGTH_ERROR,
+    _METRIC_AND_CONF_INTERVALS_NOT_SAME_LENGTH_ERROR,
+    _ERRORS_NEGATIVE_VALUE_ERROR,
+    _CONF_INTERVALS_FLIPPED_BOUNDS_ERROR,
+    _CONF_INTERVALS_MUST_BE_ARRAY
+)
+
+from .data_for_test import y_t, y_p, g_1
+
+z_score = 1.959964
+digits_of_precision = 4
+
+
+def general_wilson(p, n, digits=digits_of_precision, z=z_score):
```

Comment:
any reason these are redefined here?

Reply:
I didn't want to import from the example file, but I can do that as well

Reply:
Actually, pytest fails if I try to reference it from the example file: 
![image](https://user-images.githubusercontent.com/25800515/130264175-f7825f4e-6f15-4b4d-9e5d-470e0f638cf1.png)

Reply:
@riedgar-ms WDYT?

Reply:
I don't think that we need 'real' error calculations here.